js/chat/chat.ts

```diff
@@ -264,6 +298,9 @@ class ChatContainer extends LightElement {
 
   #appendMessage(message: Message, finalize = true): void {
     this.#removeLoadingMessage();
+    if (!this.input.disabled) {
+      this.input.disabled = true;
+    }
```

Comment:
This catches scenarios where the streaming is initiated on the server and not by the user sending a message. E.g. from the example app if you start streaming without user input:

```python
@reactive.effect
async def init_chat():
    response = chat_model.stream("Let's start our story together.")
    await chat.append_message_stream(response)
```